[DOCS] Add 7.15.0 known issue

jloleysens opened the pull request:
## Summary

Added known issue which was added to 7.14 (see revert https://github.com/elastic/kibana/pull/113441).

gchaps commented on docs/CHANGELOG.asciidoc:
```diff
@@ -73,6 +73,13 @@ For information about the {kib} 7.15.0 release, review the following information
+[%collapsible]
+====
+{kib} is unable to restore *Discover* search sessions with a relative time range. When you restore the *Discover* search session, then run a new search, {kib} displays a `Your search session is still running` message. For more information, refer to {kibana-issue}101430[#101430].
+====
```
I think this might work better:

Title: {kib} is unable to restore **Discover** search sessions with a relative time range
Description: When you restore a *Discover* search session, then run a new search, {kib} displays a `Your search session is still running` message. For more information, refer to {kibana-issue}101430[#101430]

gchaps replied:
@jloleysens The content looks good, but please add the formatting.  Put **Discover** in bold (2 places) and use code font for the message, as shown above.

jloleysens replied:
Gotcha, that was a copy-paste error :) great catch!

jloleysens commented:
Thanks for the review @gchaps ! I think I've addressed your feedback (and found we were linking to the wrong issue!). Would you mind taking another look?

gchaps commented on docs/CHANGELOG.asciidoc:
```diff
@@ -73,6 +73,13 @@ For information about the {kib} 7.15.0 release, review the following information
+[%collapsible]
+====
+*Impact* +
+When you restore a *Discover* search session, then run a new search, {kib} displays a `Your search session is still running` message. For more information, refer to {kibana-issue}112562[#112562]
```
```suggestion
When you restore a *Discover* search session, then run a new search, {kib} displays a `Your search session is still running` message. For more information, refer to {kibana-issue}112562[#112562].
```

gchaps reviewed (approved):
Add an ending period, then lgtm.
